docker-compose.yml

```diff
@@ -210,7 +210,7 @@ services:
             - JVB_BREWERY_MUC
             - MAX_BRIDGE_PARTICIPANTS
             - OCTO_BRIDGE_SELECTION_STRATEGY
-            - SENTRY_DSN="${JICOFO_SENTRY_DSN}"
+            - SENTRY_DSN="${JICOFO_SENTRY_DSN:-0}"
```

Comment:
`SENTRY_DSN` is not something I define - it's a predefined config variable of sentry. So setting it to `0` is not like it is defined in their docs.

Reply:
But if we take the fallback sentry won't be enabled, so that's ok, isn't it? I want to avoid the questions when people see warnings because they are undefined, when didn't configure any sentry related stuff.

Reply:
@saghul It's fine, you can merge it if you want.

Reply:
Alright then! I'll make sure to land this next week.